CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,76 @@
+Contributor Covenant Code of Conduct
+####################################
+
+Our Pledge
+**********
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our
+project and our community a harassment-free experience for everyone,
+regardless of age, body size, disability, ethnicity, gender identity and
+expression, level of experience, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+Also refer to our
+`Zephyr project community guidelines
+<https://www.zephyrproject.org/developers/how-to-contribute/#community-guidelines>`__
+
+Our Standards
+*************
+
+Examples of behavior that contribute to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention
+  or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or
+  electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+Our Responsibilities
+********************
+
+Project maintainers are responsible for clarifying the standards of
+acceptable behavior and are expected to take appropriate and fair
+corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit,
+or reject comments, commits, code, wiki edits, issues, and other
+contributions that are not aligned to this Code of Conduct, or to ban
+temporarily or permanently any contributor for other behaviors that they
+deem inappropriate, threatening, offensive, or harmful.
+
+Scope
+*****
+
+This Code of Conduct applies both within project spaces and in public
+spaces when an individual is representing the project or its community.
+Examples of representing a project or community include using an
+official project e-mail address, posting via an official social media
+account, or acting as an appointed representative at an online or
+offline event. Representation of a project may be further defined and
+clarified by project maintainers and `project governing bodies
+<https://www.zephyrproject.org/about/organization/>`__
+
+Reporting
+*********
+
+We strive to maintain a respectful and professional environment in the Zephyr
+Community. If you experience or witness Code of Conduct violations and wish to
+report the incident please contact us at info@zephyrproject.org.  
```

Comment:
The project "sanitycheck" has some rules about file changes that enforce good habits. One was caught here: trailing white space is not allowed, there are a couple of spaces at the end of this line you should delete, reported by https://app.shippable.com/github/zephyrproject-rtos/zephyr/runs/23650/5/tests